Hogajama/hogajama-domain/src/test/java/com/gepardec/hogarama/domain/sensor/SensorNormalizerTest.java

```diff
@@ -53,6 +60,9 @@ public SensorNormalizerTest(Double expectedValue, String type, Double value) {
 	@Before
 	public void setUpMethod() throws Exception {
 		sn = new SensorNormalizer();
+		sensorCache = mock(SensorCache.class);
+		Mockito.when(sensorCache.getByDeviceId(DEVICE_ID)).thenReturn(Optional.empty());
+		sn.setSensorCache(sensorCache);
```

Comment:
is there any reason why you use a setter and the standard junit runner instead of @InjectMocks with MockitoJUnitRunner? Using the MockitoJUnitRunner would make the setter obsolet

Reply:
As discussed, i don't know how to combine MockitoJUnitRunner with Parameterized.